### PR TITLE
fix(developer): remove unused keyboard info compiler messages

### PR DESCRIPTION
**Pull request by markcsinclair**
Removed five unused keyboard info compiler messages:
- Error_FileIsNotValid:
- Warn_MetadataFieldInconsistent
- Hint_OutputValidation
- Warn_OutputValidation
- Error_OutputValidation

@keymanapp-test-bot skip

### CODE REVIEW

File: developer/src/kmc-keyboard-info/src/keyboard-info-compiler-messages.ts [hunk 1]
```diff
@@ -17,14 +17,6 @@ export class KeyboardInfoCompilerMessages {
...
   static Error_FileDoesNotExist = (o:{filename: string}) => m(this.ERROR_FileDoesNotExist, `File ${def(o.filename)} does not exist.`);
 
-  static ERROR_FileIsNotValid = SevError | 0x0003;
-  static Error_FileIsNotValid = (o:{filename: string; e: any}) => m(this.ERROR_FileIsNotValid,
-    `File ${def(o.filename)} could not be parsed: ${(o.e ?? 'unknown error').toString()}.`);
-
-  static WARN_MetadataFieldInconsistent = SevWarn | 0x0004;
-  static Warn_MetadataFieldInconsistent = (o:{field:string, value:any, expected:any}) => m(this.WARN_MetadataFieldInconsistent,
-    `Warning: field ${def(o.field)} value "${def(o.value)}" does not match "${def(o.expected)}" found in source file metadata.`);
-
```
Comment (mcdurdin) [inline developer/src/kmc-keyboard-info/src/keyboard-info-compiler-messages.ts:27]: Might be good to include a comment

```
// 0x0003 unused, available for future messages
// 0x0004 unused, available for future messages
```

File: developer/src/kmc-keyboard-info/src/keyboard-info-compiler-messages.ts [hunk 2]
```diff
@@ -49,18 +41,6 @@ export class KeyboardInfoCompilerMessages {
...
-    `Validating output: ${def(o.message)}.`);
-
-  static WARN_OutputValidation = SevWarn | 0x000C;
-  static Warn_OutputValidation = (o:{message: any}) => m(this.WARN_OutputValidation,
-    `Validating output: ${def(o.message)}.`);
-
-  static ERROR_OutputValidation = SevError | 0x000D;
-  static Error_OutputValidation = (o:{message: any}) => m(this.ERROR_OutputValidation,
-    `Validating output: ${def(o.message)}.`);
-
```
Comment (mcdurdin) [inline developer/src/kmc-keyboard-info/src/keyboard-info-compiler-messages.ts:63]: Might be good to include a comment

```
// 0x000B unused, available for future messages
// 0x000C unused, available for future messages
// 0x000D unused, available for future messages
```

**Review by mcdurdin — APPROVED** [review]
LGTM. If you don't mind adding those comments in for future maintainers, that'd be great.

File: developer/src/kmc-keyboard-info/src/keyboard-info-compiler-messages.ts (hunk 1 above)
Comment (markcsinclair) [reply developer/src/kmc-keyboard-info/src/keyboard-info-compiler-messages.ts:27]: Agreed

File: developer/src/kmc-keyboard-info/src/keyboard-info-compiler-messages.ts (hunk 2 above)
Comment (markcsinclair) [reply developer/src/kmc-keyboard-info/src/keyboard-info-compiler-messages.ts:63]: Agreed

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.292-beta](https://keyman.com/downloads/releases/beta/17.0.292)

**Pull request merged**